### ISSUE TITLE
Stop rendering  tourism=attraction with name label without additional qualifiers

### ISSUE THREAD

**Issue by imagico** [gravitystorm/openstreetmap-carto#5011]
We currently render `tourism=attraction` with a name label in `amenity-points`.  This rendering has a long history of changes - see for example #941, #1063, #1257, #1824, #3603.

It has, however, never been semantically very meaningful because `tourism=attraction` is not used typically as a primary tag in a meaningful way but rather as a secondary tag indicating a feature (like an amenity, a building or a historic site of some kind) has a touristic significance.  In those case it would be of more value to display the type of feature (amenity, building, historic site etc.) in rendering and possibly vary the rendering based on the touristic significance rather than displaying the name of a generic touristic attraction and not showing the main classification (since the generic label will block that).  Many uses of the tag in combination with a name and without additional qualifiers also do not contain a proper name in the name tag but rather a classification or other types of free form labels.

Instead rendering more specific touristic attractions with a well defined primary tag could be advisable (like #3550, #3775).

**Comment by dch0ph** [comment]
Agree. As `tourism=attraction` can be slapped onto anything, it can't be coherently rendered.

Often it seems to be used as tagging-for-the-render to get a label to [appear](https://www.openstreetmap.org/#map=19/54.375788/-3.050677):

![image](https://github.com/user-attachments/assets/34d91bf9-3afa-40b3-adde-0bf9089b0b93)

[No primary tags. No particular reason why this should be highlighted for tourists either.]
